tests/acceptance/premium/test_group_push_rules.py

```diff
@@ -0,0 +1,46 @@
+import pytest
+from tests.acceptance import run_gitlabform
+
+pytestmark = pytest.mark.requires_license
+
+
+class TestGroupPushRules:
+    def test__create_group_push_rules(self, group):
+        config = f"""
+        projects_and_groups:
+          {group.full_path}/*:
+            group_push_rules:
+              commit_message_regex: ''
+              member_check: false
+              commit_committer_check: true
+              commit_committer_name_check: true
+        """
+
+        run_gitlabform(config, group)
+
+        push_rules = group.pushrules.get()
+        assert push_rules.commit_message_regex == ""
+        assert push_rules.member_check is False
+        assert push_rules.commit_committer_check is True
+        assert push_rules.commit_committer_name_check is True
+
+    def test__edit_group_push_rules(self, group):
+        self.test__create_group_push_rules(group)
```

Comment:
Don't need to do this. You can delete this line.

The `group` parameter is a pytest fixture. If you check this fixture in `conftest.py`, you'll notice that this fixture is set to `class` scope. That means, this group is available for the entire duration of the `TestGroupPushRules` class. As a result, when the first/previous test case runs, it already created/configured the group push rule for you. That's why you don't need to call the previous test case again.

Hope that makes sense.